alma/views/templates/hook/displayProductActions.tpl

```diff
@@ -21,6 +21,6 @@
  * @license   https://opensource.org/licenses/MIT The MIT License
  *}
 <div class="alma-widget-insurance" id="alma-widget-insurance-product-page" data-alma-insurance-settings="{$settingsInsurance}" style="height: 100%" >
-    <div id="product-details" class="js-product-details" data-product="{$product.embedded_attributes|json_encode}" style="display:none;"></div>
```

Comment:
I had used a custom id `#alma-product-details` also, but I saw the `js-product-details` event rename it. I think this change create an other issue with an other merchant. Maybe we will need to recreate our own reload product-details ?